pandas/tests/groupby/test_apply.py

```diff
@@ -63,15 +63,8 @@ def test_apply_trivial():
     tm.assert_frame_equal(result, expected)
 
 
-@pytest.mark.xfail(
-    reason="GH#20066; function passed into apply "
-    "returns a DataFrame with the same index "
-    "as the one to create GroupBy object."
-)
 def test_apply_trivial_fail():
-    # GH 20066
-    # trivial apply fails if the constant dataframe has the same index
-    # with the one used to create GroupBy object.
+    # GH 35441
```

Comment:
I think this should be left as 20066. That issue explains why the test is here, not this PR.

Reply:
Done.